sematic/db/models/factories.py

```diff
@@ -153,32 +148,22 @@ def make_artifact(
     json_summary = get_json_encodable_summary(value, type_)
 
     artifact = Artifact(
-        id=_get_value_sha1_digest(
-            value_serialization, type_serialization, json_summary
-        ),
-        json_summary=_fix_nan_inf(
-            json.dumps(json_summary, sort_keys=True, default=str)
-        ),
+        id=_get_value_sha1_digest(value_serialization, type_serialization, json_summary),
+        json_summary=_fix_nan_inf(json.dumps(json_summary, sort_keys=True, default=str)),
         type_serialization=json.dumps(type_serialization, sort_keys=True),
         created_at=datetime.datetime.utcnow(),
         updated_at=datetime.datetime.utcnow(),
     )
 
-    if storage is not None:
-        storage.set(
-            _make_artifact_storage_key(artifact),
-            json.dumps(value_serialization, sort_keys=True).encode("utf-8"),
-        )
+    payload = json.dumps(value_serialization, sort_keys=True).encode("utf-8")
 
-    return artifact
+    return artifact, payload
 
 
-def get_artifact_value(artifact: Artifact, storage: Storage) -> Any:
+def deserialize_artifact_value(artifact: Artifact, payload: bytes) -> Any:
     """
     Fetch artifact serialization from storage and deserialize.
```

Comment:
This docstring is not longer accurate; please update.

Reply:
Done.